pkg/stats/monitor.go

```diff
@@ -60,20 +60,23 @@ const (
 type Monitor struct {
 	nodeID string
 
-	inviteReqRaw    prometheus.Counter
-	inviteReq       *prometheus.CounterVec
-	inviteAccept    *prometheus.CounterVec
-	inviteErr       *prometheus.CounterVec
-	callsActive     *prometheus.GaugeVec
-	callsTerminated *prometheus.CounterVec
-	packetsRTP      *prometheus.CounterVec
-	durSession      *prometheus.HistogramVec
-	durCall         *prometheus.HistogramVec
-	durJoin         *prometheus.HistogramVec
-	durCheck        *prometheus.HistogramVec
-	cpuLoad         prometheus.Gauge
-	sdpSize         *prometheus.HistogramVec
-	nodeAvailable   prometheus.GaugeFunc
+	inviteReqRaw       prometheus.Counter
+	inviteReq          *prometheus.CounterVec
+	inviteAccept       *prometheus.CounterVec
+	inviteErr          *prometheus.CounterVec
+	callsActive        *prometheus.GaugeVec
+	callsTerminated    *prometheus.CounterVec
+	packetsRTP         *prometheus.CounterVec
+	durSession         *prometheus.HistogramVec
+	durCall            *prometheus.HistogramVec
+	durJoin            *prometheus.HistogramVec
+	durCheck           *prometheus.HistogramVec
+	cpuLoad            prometheus.Gauge
+	sdpSize            *prometheus.HistogramVec
+	nodeAvailable      prometheus.GaugeFunc
+	transfersTotal     *prometheus.CounterVec
```

Comment:
We could also add active transfers metric as a gauge and inc/dec it in the existing helper functions.